support-services/src/main/scala/com/gu/support/catalog/AwsS3ClientJson.scala

```diff
@@ -0,0 +1,26 @@
+package com.gu.support.catalog
+
+import com.amazonaws.services.s3.AmazonS3URI
+import com.gu.aws.AwsS3Client
+import com.typesafe.scalalogging.LazyLogging
+import io.circe._
+import io.circe.parser._
+
+import scala.util.{Failure, Success}
+
+object AwsS3ClientJson extends LazyLogging{
```

Comment:
pulled out from the original S3 client, otherwise we would have needed circe as a dependency of the module-aws.